spec/requests/moderate_stories_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+
+RSpec.describe "ModerateStories", type: :request do
```

Comment:
Because moderator functionality is so much less used, this can be one test. Load the form for a story, change the title, submit the form, then assert that it was redirected to the story and that the new title appears. We'll only add more tests if things become unreliable, but this is a clear refactor of very-well-used code so I don't worry that it needs a lot of tests.